modern/src/runtime/GuiObject.ts

```diff
@@ -136,29 +137,41 @@ class GuiObject extends MakiObject {
   }
 
   getheight(): number {
-    // TODO
-    // I don't know how it gets calculated exactly, but if a node has a minimum
-    // and maximum h, but no h, getwidth still returns a value, return min for now
-    // TODO: Need to return actual current width which can be based on underlying image
-    // CornerAmp has a calls like: obj.resize(newX, newY, obj.getwidth(), obj.getheight()),
-    // but the object has no width/height set, so it needs to return the width/height
-    // that potentially uses the image size like we do in the renderer.
-    // This is made more complicated because images depend on the current state and are
-    // potentially different attributes for different MakiObjects
-    return Number(this.attributes.h) || Number(this.attributes.minimum_h) || 0;
+    if (this.attributes.h !== undefined) {
+      return Number(this.attributes.h);
```

Comment:
I removed returning `minimum_h` because in retrospect I think that was just a hack for cases when we really should have been using the underlying image size